Makefile

```diff
@@ -31,7 +31,7 @@ THRIFT_GEN_DIR=thrift-gen
 PASS=$(shell printf "\033[32mPASS\033[0m")
 FAIL=$(shell printf "\033[31mFAIL\033[0m")
 COLORIZE=$(SED) ''/PASS/s//$(PASS)/'' | $(SED) ''/FAIL/s//$(FAIL)/''
-DOCKER_NAMESPACE?=$(USER)
+DOCKER_NAMESPACE?=jaegertracing
```

Comment:
Setting the repository to the correct one. I added user input to docker push to avoid unwanted pushes.

Reply:
This was on purpose, so that we don't push accidentally to the *official* repository. Just pass `DOCKER_NAMESPACE` before the `make` command to override it:

```
DOCKER_NAMESPACE=jaegertracing make
```

Or, like this:
https://github.com/jaegertracing/jaeger/blob/f0193c63aefb9be980b639ea59d612ad47b78ba7/travis/build-docker-images.sh#L23

Reply:
>This was on purpose, so that we don't push accidentally to the official repository. 

I know, because of this I have added user input/verification before docker push.

This change is because now xdock uses only docker images so it has to build. We could use different image names/repository in docker-compose but that seems more odd.

Reply:
This should work `CONTINUE=y make docker-push`